container/container-core/src/main/java/org/talend/sdk/component/classloader/ConfigurableClassLoader.java

```diff
@@ -108,6 +108,10 @@ public class ConfigurableClassLoader extends URLClassLoader {
 
     private final URLClassLoader classLoaderFromClasspath;
 
+    @Getter
+    private final List<String> cacheableClasses =
```

Comment:
when this parameter not set, this will return a array with lenght 1 : [""], is not expected in my view.